compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -1230,51 +1206,48 @@ pub trait PrettyPrinter<'tcx>: Printer<'tcx> + fmt::Write {
     /// traits map or fn_traits map, depending on if the trait is in the Fn* family of traits.
     fn insert_trait_and_projection(
```

Comment:
I'm pretty disappointed with how sensitive to bound var renumbering this is... but 🤷